frontend/src/components/login/LoginManager.tsx

```diff
@@ -29,10 +30,16 @@ export const LoginManager: React.FunctionComponent = ({ children }) => {
   const [Admin, setAdmin] = useState(false);
 
   function logIn(username: string, password: string) {
-    setUsername(username);
-    setPassword(password);
-    setLoggedIn(true);
-    setAdmin(true);
+    authenticateLogin(username, password).then((didLogin) => {
+      if (didLogin) {
+        setUsername(username);
+        setPassword(password);
+        setLoggedIn(true);
+        setAdmin(true);
+      } else {
+        console.log("User could not be authenticated.");
+      }
+    });
```

Comment:
Oops! Sorry, I should have caught this before, but there should only be changes to `internalApiClient.ts` in this ticket. Could you remove those changes from here, and bring them forward into the new branch you're working on? That's potentially git-tricky so please get me to pair on it if you're stuck for even a second